doc/faq.rst

```diff
@@ -58,6 +58,15 @@ fundamentally multi-dimensional. If your data is unstructured or
 one-dimensional, stick with pandas.
 
 
+Gotchas
+-------
+
+xarray tries hard to be self-consistent. Operations on xarray objects return
+other xarray objects. In particular, operations like `mean` or `sum` - even
+when applied to all axes - will return xarray objects. Constructing `pandas`
+objects sometimes require explicit casting.
```

Comment:
I don't quite understand what this last comment means, maybe an example would help?

Reply:
I'll add something similar to my reply to @fmaussion above